src/clarity-angular/datagrid/render/main-renderer.ts

```diff
@@ -53,6 +69,23 @@ export class DatagridMainRenderer implements AfterContentInit, AfterViewInit, On
         });
     }
 
+    private _heightSet: boolean = false;
+
+    /**
+     * Computes the height of the datagrid.
+     *
+     * NOTE: We had to choose to set the height instead of the min-height because
+     * IE 11 requires the height on the parent for the children flex grow/shrink properties to work.
+     * When we used min-height, 1 1 auto doesn't used to work in IE11 :-(
+     * But this doesn't affect the fix. It works in both fixed & variable height datagrids.
+     *
+     * Refer: http://stackoverflow.com/questions/24396205/flex-grow-not-working-in-internet-explorer-11-0
+     */
+    private computeDatagridHeight() {
+        let value: number = this.domAdapter.computedHeight(this.el.nativeElement);
+        this.renderer.setElementStyle(this.el.nativeElement, "height", value + "px");
```

Comment:
Just FYI, @youdz and I discussed to keep this as `height` because of the IE11 issues.